README.md

```diff
@@ -1 +1,206 @@
-# resort-booking-backend-app
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h1><b> 🏖 Resort Booking 🏞 </b></h1>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [💸 Resort Booking ](#-resort-booking-)
+- [🛠 Built With ](#-built-with-)
+  - [📌 Tech Stack ](#-tech-stack-)
+- [🎲 Key Features ](#-key-features-)
+- [🚀 Live Demo](#-live-demo-)
+- [💻 Getting Started ](#-getting-started-)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors ](#-authors-)
+- [🔭 Future Features ](#-future-features-)
+- [🤝 Contributing ](#-contributing-)
+- [⭐️ Show your support ](#️-show-your-support-)
+- [🙏 Acknowledgments ](#-acknowledgments-)
+- [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# Resort Booking <a name="about-project"></a>
+
+**Resort Booking** is a web application that offers a diverse selection of luxurious accommodations and amenities. Find your slice of paradise, effortlessly plan your trip, and make memories that last a lifetime with ResortHub.
+
+
+# 🛠 Built With <a name="built-with"></a>
+
+## 📌 Tech Stack <a name="tech-stack"></a>
+
+| Domain | Tech Stack |
+|--------|--------|
+| Server | <a href="https://rubyonrails.org/">Ruby on Rails</a> |
+| Database | <a href="https://www.postgresql.org/">PostgreSQL</a> |
+| Testing Framework | <a href="https://github.com/rspec/rspec-rails">RSpec Rails</a> |
+| Authentification | <a href="https://github.com/heartcombo/devise#getting-started">Devise</a> |
+| Authorization | <a href="https://jwt.io/introduction">JWT</a> | 
+
+<!-- Features -->
+
+# 🎲 Key Features <a name="key-features"></a>
+
+- Create a new account
+- Log in to an existing account
+- Sign out from a session
+- Choose from a variety of beatiful resorts
+- Book a resort
+- Add a resort
+- Delete a resort
+
+# 🚀 Live Demo <a name="live-demo"></a>
+
+👉 [Not Available](https://....)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- GETTING STARTED -->
+
+# 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+## Prerequisites
+
+In order to run this project you need:
+
+- **Ruby 3.2.2** installed
+
+- **Rails** installed
+
+Run this command to install **Rails**
+
+```sh
+ gem install rails
+```
+## Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone https://github.com/Leeoasis/resort-booking-backend-app.git
+```
+
+To install dependencies, run:
+
+```sh
+bundle install
+```
+
+**Set up the database** <br>
+
+To create the database, run:
+
+```sh
+rails db:create
+```
+To create/update the schema, run:
+
+```sh
+rails db:migrate
+```
+
+## Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
+## Run tests
+
+To run tests, run the following command:
+
+```sh
+  rspec
+```
```

Comment:
- [ ]  You are using Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍